lib/main.dart

```diff
@@ -1,21 +1,37 @@
+import 'src/styles/dimensions.dart';
 import 'package:flutter/material.dart';
-import 'src/screen/ui_dice.dart';
+import 'src/bloc/dice_bloc.dart';
+import 'src/screen/dice_screen.dart';
 
 void main() {
   runApp(DiceRoller());
 }
 
-class DiceRoller extends StatelessWidget {
-  // This widget is the root of your application.
+class DiceRoller extends StatefulWidget {
+  @override
+  _DiceRollerState createState() => _DiceRollerState();
+}
+
+class _DiceRollerState extends State<DiceRoller> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  final DiceBloc diceBloc = DiceBloc();
+
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Roll the Dice App',
+      title: Dimension.mainAppTitle,
```

Comment:
it's fine, but maybe you need to rename the "Dimensions files" to "Constants, maybe.

Reply:
done